### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by nouiz**

### CODE REVIEW

File: theano/sparse/tests/test_basic.py [hunk 1]
```diff
@@ -2105,7 +2105,6 @@ def op_with_fixed_index(x):
 
     def test_GetItem2D(self):
         scipy_ver = [int(n) for n in scipy.__version__.split('.')[:2]]
-        assert scipy_ver >= [0, 11]
```
Comment (abergeron) [inline theano/sparse/tests/test_basic.py]: This was added because we were unsure of the behaviour with older version.  Did you make sure it's ok?

**Comment by abergeron** [comment]
Apart from the scipy comment, I'm ok whit this.

File: theano/sparse/tests/test_basic.py (hunk 1 above)
Comment (nouiz) [inline theano/sparse/tests/test_basic.py]: This made our daily buildbot fail this test as it use scipy 0.9. I ran the test with that version of scipy and they passed. So I think it is ok.

**Comment by nouiz** [comment]
As @abergeron told he was ok and I responded to his comment, I merge now to get this in the master for next buildbot.

**Pull request merged**